modules/cert-manager-configure-cloud-credentials-aws-non-sts.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on AWS
+
+To configure the cloud credentials for the {cert-manager-operator} on the AWS cluster you must create a `CredentialsRequest` object and allow the Cloud Credential Operator to generate the cloud credentials secret.
+
+.Prerequisites
+
+* The {cert-manager-operator} 1.11.1 or later is installed.
```

Comment:
Similar to https://github.com/openshift/openshift-docs/pull/60433 add one more prerequisite about CCO mode.